easybuild/framework/easyblock.py

```diff
@@ -1585,7 +1607,7 @@ def sanity_check_step(self, custom_paths=None, custom_commands=None, extension=F
                 found = False
                 for name in xs:
                     path = os.path.join(self.installdir, name)
-                    if os.path.exists(path):
+                    if check_fn(path):
```

Comment:
this is indeed a bug, this fix is also included in #1388 (which doesn't mean it shouldn't be included here)

maybe we should make this a separate PR though, to make the bug fix stand out; it may have some fallout, i.e. causing sanity checks to fail on empty directories where they didn't before (they should have though)